marklogic-spark-connector/src/main/java/com/marklogic/spark/writer/DocBuilder.java

```diff
@@ -155,9 +159,19 @@ private DocumentWriteOperation buildMainDocument(DocumentInputs inputs) {
         final DocumentMetadataHandle metadataFromRow = inputs.getInitialMetadata();
 
         AbstractWriteHandle content = inputs.getContent();
+        Format sourceDocumentFormat = Util.determineSourceDocumentFormat(content, sourceUri);
+        if (sourceDocumentFormat == null) {
```

Comment:
Remove this logging, it's not relevant here. 

But it's worth having this logging below, if the classification response is not null and if the format is neither XML nor JSON - e.g. "Cannot add classification to document with URI {}; document is neither JSON nor XML." That will typically be due to a misconfiguration by the user (or by having XML/JSON documents with unrecognized extensions) and a warning is appropriate so the user isn't wondering why their documents don't have any classification data (even though it took forever to generate that classification data).